projects/plugins/jetpack/extensions/blocks/simple-payments/edit.js

```diff
@@ -616,7 +616,9 @@ const mapSelectToProps = withSelect( ( select, props ) => {
 
 	return {
 		hasPublishAction: !! get( post, [ '_links', 'wp:action-publish' ] ),
-		isSaving: !! isSavingPost(),
+		isSaving: __experimentalGetDirtyEntityRecords().some( record =>
```

Comment:
nitpick: I think it's a common pattern to use an alias for _experimental functions when importing them.

Reply:
Are there any further conventions around experimental functions? Looking at discussions, this one has been labelled experimental for some time.

Reply:
No idea, maybe @Copons knows other?

Reply:
I think he thinks it's fine.